yarn.lock

```diff
@@ -544,19 +855,19 @@
     prettier "^2.1.2"
 
 "@pagopa/express-azure-functions@^2.0.0":
-  version "2.0.0"
-  resolved "https://registry.yarnpkg.com/@pagopa/express-azure-functions/-/express-azure-functions-2.0.0.tgz#eb52a0b997d931c1509372e2a9bea22a8ca85c17"
-  integrity sha512-IFZqtk0e2sfkMZIxYqPORzxcKRkbIrVJesR6eMLNwzh1rA4bl2uh9ZHk1m55LNq4ZmaxREDu+1JcGlIaZQgKNQ==
+  version "2.1.1"
+  resolved "https://registry.yarnpkg.com/@pagopa/express-azure-functions/-/express-azure-functions-2.1.1.tgz#ac6aa6d8c8d82b64e0db19ccadd2e89cc35ea725"
+  integrity sha512-x2kBGS6JKYqVp7PS9QiBc2AWIOPBObHwlaheJmxZJacnxmLdXHobomTzN2UJMxu55fxMd7xIxqM8VGMQxSzqiw==
 
 "@pagopa/io-functions-commons@^21.0.0":
-  version "21.5.2"
```

Comment:
why this version changed? 
there is a not-committed package.json or it changed due to ^ ?

in the former case, could be better to explicitly update the commons version in the package.json?

Reply:
Probably something happened during the yarn.lock regeneration... anyway i have updated now `fn-commons` so it is based on Node 18 here [52ea2c7](https://github.com/pagopa/io-functions-assets/pull/77/commits/52ea2c7e0c0ed8f5c5e4a33d4c7553bbec70f9a3)